apps/serverless/src/handler.ts

```diff
@@ -0,0 +1,10 @@
+import type { Handler } from "serverless-http";
+import serverlessHttp from "serverless-http";
+
+import { app } from "@/app";
```

Comment:
Importing "locally" is not enough to validate that monorepo aliases work. Please import `@nighttrax/foo` and return `meaningOfLife` in `requestListener` instead.

Reply:
fixed